build.sbt

```diff
@@ -22,7 +22,7 @@ val dependencies = Seq(
   ws,
   hmrc                %% s"bootstrap-backend-$playVersion"  % "5.12.0",
   hmrc                %% "domain"                           % s"6.2.0-$playVersion",
-  hmrc                %% "simple-reactivemongo"             % s"8.0.0-$playVersion",
+  "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"               %"0.68.0",
```

Comment:
NABD but i think 0.68.0 can be stored in a val and hence will need to be changed at one place only in future

Reply:
it doesn’t really make that big difference since it used only twice, but i've made the changes